pkg/errors/errors.go

```diff
@@ -22,13 +22,25 @@ func NewEgressURLError(failure string) error {
 }
 
 type GenericError struct {
-	e string
+	message string
```

Comment:
if we want to change `e` -> `message`, we should perform this to all EgressURLError, GenericError, or any other err we defined, in terms of standardization

Reply:
Will do. I wasn't sure about that.